monkey-runner/build.gradle

```diff
@@ -0,0 +1,17 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.1.2'
```

Comment:
It would be good to pull out all these dependencies in a separate file in order to easy their maintenance.